packages/plugins/apollo-inline-trace/src/index.ts

```diff
@@ -2,6 +2,7 @@ import { isAsyncIterable, Plugin, YogaInitialContext } from 'graphql-yoga'
 import { GraphQLError, ResponsePath } from 'graphql'
 import ApolloReportingProtobuf from 'apollo-reporting-protobuf'
 import { btoa } from '@whatwg-node/fetch'
+import { useOnResolver } from './on-resolver.js'
```

Comment:
Envelop has a plugin now, use it instead: https://github.com/n1ru4l/envelop/pull/1500.